resources/views/admin/pages/discount-group/layout/list.blade.php

```diff
@@ -43,6 +43,12 @@ class="col-lg-offset-1 col-lg-10 col-md-offset-0 col-md-12 col-sm-offset-0 col-s
                    href="{{ route('admin.discount-group.product-filter.index', $discount_group) }}">
                     <i class="fa fa-cubes"></i>
                 </a>
+
+                @if((!$discount_group->is_active)AND(((Carbon\Carbon::now())->diffInDays($discount_group->updated_at))>7))
```

Comment:
Also you can add an accessor method (for DiscountGroup) named:
function getCanDeleteAttribute() {
!$discount_group->is_active)AND(((Carbon\Carbon::now())->diffInDays($discount_group->updated_at)
}
So that you can get the property @if($discount_group->can_delete)